### DISCUSSION TITLE
[Code view beta] Add files tree view also to root repo

### DISCUSSION THREAD

**Discussion by maicol07** [community/community#38758] [Repositories]
### Select Topic Area

Product Feedback

### Body

Currently the new files tree view only appears when opening a file or a directory. It would be great to have the files tree view also in the root directory.

**Comment by artursouza** [comment]
I agree. It is really annoying to switch from old mode to tree view after clicking on a file or directory. At least, offer this as a configuration to the authenticated user since some people might like the clean view when landing on a repository's root.

**Comment by ossanna16** [comment]
Hi there @maicol07 and welcome to our community! Thank you for the feedback!

**Comment by liby** [comment]
You can press <kbd>T</kbd> key to [open the file tree view](https://docs.github.com/en/get-started/using-github/keyboard-shortcuts#source-code-browsing:~:text=Activates%20the%20file%20finder).

**Reply by maicol07** [reply]
Thank you! However this doesn't show the repo information. @ossanna16 is there any way to add them to the repo root when using file tree view? Also an option in settings to activate automatically the file tree view on a repo root would be great!

**Reply by liby** [reply]
> Thank you! However this doesn't show the repo information. @ossanna16 is there any way to add them to the repo root when using file tree view? Also an option in settings to activate automatically the file tree view on a repo root would be great!

It doesn't seem to work every time.

https://user-images.githubusercontent.com/38807139/202183962-0ff8189a-a082-448f-a0a2-35bfef05a76f.mp4
